lienzo-core/src/main/java/com/ait/lienzo/client/core/shape/AbstractMultiPathPartShape.java

```diff
@@ -380,7 +380,7 @@ public DefaultMultiPathShapeHandleFactory(NFastArrayList<PathPartList> listOfPat
         @Override
         public Map<ControlHandleType, IControlHandleList> getControlHandles(List<ControlHandleType> types) {
             if ((null == types) || (types.isEmpty())) {
-                return null;
+                return new HashMap<>();
```

Comment:
Hi @jomarko,

did you check all `getControlHandles` method invocations? If I correct some of them already relies on `null`s so this small change can lead to further issues.

Reply:
agree, good point @hasys , either we stay with `null` or usages of this should be at least checked, and if applies, refactored. I would recommend @jomarko to stay with `null`....